modular_skyrat/modules/morenarcotics/code/quaalude.dm

```diff
@@ -13,39 +14,52 @@
 	taste_description = "lemons"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
-/datum/reagent/drug/quaalude/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
-	var/high_message = pick("You feel relaxed.", "You feel like you're on the moon.", "You feel like you could walk 20 miles for a quaalude.")
-	if(DT_PROB(2.5, delta_time))
-		to_chat(M, span_notice("[high_message]"))
-	if(M.hud_used!=null)
-		var/atom/movable/plane_master_controller/game_plane_master_controller = M.hud_used.plane_master_controllers[PLANE_MASTERS_GAME]
+
+/datum/reagent/drug/quaalude/on_mob_metabolize(mob/living/carbon/affected_carbon)
+	if(affected_carbon.hud_used)
+		var/atom/movable/plane_master_controller/game_plane_master_controller = affected_carbon.hud_used.plane_master_controllers[PLANE_MASTERS_GAME]
 		game_plane_master_controller.add_filter("quaalude_wave", 10, wave_filter(300, 300, 3, 0, WAVE_SIDEWAYS))
-	M.set_drugginess(1 MINUTES * REM * delta_time)
-	M.adjust_slurring(1 MINUTES)
-	M.set_dizzy_if_lower(5 * REM * delta_time * 2 SECONDS)
-	M.adjustStaminaLoss(-5 * REM * delta_time, 0)
+
+
+/datum/reagent/drug/quaalude/on_mob_life(mob/living/carbon/affected_carbon, delta_time, times_fired)
+	if(DT_PROB(2.5, delta_time))
+		var/high_message = pick("You feel relaxed.", "You feel like you're on the moon.", "You feel like you could walk 20 miles for a quaalude.")
+		to_chat(affected_carbon, span_notice("[high_message]"))
```

Comment:
I'm not sure, to be fair. Also, I just saw something in there that I definitely have to change, thanks for pointing it out.

Reply:
I don't think there's really any other span that's used for drugs currently, although I could be wrong since I haven't used drugs much myself, but I feel like that's the case.